packages/async/package.json

```diff
@@ -0,0 +1,79 @@
+{
+    "name": "@solana/async",
+    "version": "2.0.0-rc.1",
+    "description": "Helpers for working with async values",
+    "exports": {
+        "browser": {
```

Comment:
Probably won't matter for this package but we should add `edge-light` and `workerd` exports for consistency. I've merged that PR now